### DISCUSSION TITLE
The output form of  forecast() in  predict.py

### DISCUSSION THREAD

**Discussion by YE-JR** [PaddlePaddle/PaddleSpatial#112] [BaiduKDDCup2022] [answered]
Can you provide a detailed description of the output form of the forecast ()

**Comment by ramber1836** [comment, marked as answer]
The predictions as a numpy ndarray with the shape of 134 * 288 * 1

Regards,
Xinjiang Lu

**Comment by YE-JR** [comment]
Thank you for your reply！
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PaddlePaddle/PaddleSpatial"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月28日(星期六) 中午11:55
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [PaddlePaddle/PaddleSpatial] The output form of  forecast() in  predict.py (Discussion #112)





 The predictions as a numpy ndarray with the shape of 134 * 288 * 1
 
 Regards,
 Xinjiang Lu
 
 
 On Fri, May 27, 2022 at 5:12 PM YE-JR ***@***.***&gt; wrote:
 
 &gt; Can you provide a detailed description of the output form of the forecast
 &gt; ()
 &gt;
 &gt; —
 &gt; Reply to this email directly, view it on GitHub
 &gt; <https://github.com/PaddlePaddle/PaddleSpatial/discussions/112&gt;, or
 &gt; unsubscribe
 &gt; <https://github.com/notifications/unsubscribe-auth/AEVHY36E5JVIGCQH5DO7QWDVMCGYBANCNFSM5XDT4WFA&gt;
 &gt; .
 &gt; You are receiving this because you are subscribed to this thread.Message
 &gt; ID: ***@***.***&gt;
 &gt;
